wsgi/playdoh.wsgi

```diff
@@ -16,22 +15,14 @@ if newrelic:
     else:
         newrelic = False
 
-os.environ.setdefault('CELERY_LOADER', 'django')
-# NOTE: you can also set DJANGO_SETTINGS_MODULE in your environment to override
-# the default value in manage.py
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'bedrock.settings')
 
-# Add the app dir to the python path so we can import manage.
-wsgidir = os.path.dirname(__file__)
-site.addsitedir(os.path.abspath(os.path.join(wsgidir, '../')))
-
-# manage adds /apps, /lib, and /vendor to the Python path.
-import manage
-
+# must be imported after env var is set above.
 from django.core.wsgi import get_wsgi_application
 from whitenoise.django import DjangoWhiteNoise
 
-application = DjangoWhiteNoise(get_wsgi_application())
+application = get_wsgi_application()
+application = DjangoWhiteNoise(application)
```

Comment:
only reason I like it this way and left it was because I was debugging this file and it helped to be able to easily comment out whitenoise for a bit to make sure it wasn't causing the issue. So I like having them on separate lines just for how easy it makes disabling individual middleware.